.github/workflows/tests.yml

```diff
@@ -1,6 +1,12 @@
 name: Tests
 
-on: [push]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
After this change the test don't run on push to branches without pull request

Is there a way to have both for the pull request but also for branches while developing

Reply:
I couldn't find one easily. Problem is, if you don't have this it will run tests twice for branches in the same repo. Quick hack is to remove this line in your branch and add it back before making the PR.

Reply:
Nice trick 👍 

_I'll try removing the code while pushing to a branch and remove it once the pull request is open_